docker-compose.lando.yml

```diff
@@ -12,7 +12,6 @@ services:
     command: bash -c "
       lando generate_version_file &&
       lando collectstatic --clear --no-input &&
-      lando migrate &&
```

Comment:
Might make sense to add a small mention in the commit message as to why this was removed (i.e., it's taken care of in `setup_dev`.) I am wondering if we should also move `collectstatic` to `setup_dev` as well, or possibly moving both of those out of there and into here, for consistency.

Reply:
Yeah, I'd like to reduce the `command`. I was about to do it here, but realised this required new changes to lando, so this'll have to wait. 

I think we could also look at adding an `entrypoint` instead, which could benefit all containers.

Bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1940601